### PR TITLE
Update RSE Team training info

### PR DESCRIPTION
**Pull request by bobturneruk**
Hopefully better. For currently scheduled courses, I've linked the RSE Team hope page which is where folks can see the current dates and link through for signup. Could do deeper links?

### CODE REVIEW

**Comment by bobturneruk** [comment]
@jkwmoore - please can you review this?

**Comment by jkwmoore** [comment]
FYI I've not tested if the topics will work with a space (i.e. the filtering), note that tags are also tied to the javascript filtering so changing existing tags will break that.

**Comment by bobturneruk** [comment]
Ta, I'll have another look.

**Comment by jkwmoore** [comment]
See this bit: https://github.com/rcgsheffield/TUoS-RIT-training-resources/blob/master/assets/js/search.js

**Comment by bobturneruk** [comment]
Hi @jkwmoore - I've hopefully got rid of any tags / topics that might cause problems, for now. I'd need you to enable the workflow to test if this is OK.

**Comment by bobturneruk** [comment]
Oh seems like it builds, at least!

File: _data/links/RSE_Courses.yml [hunk 1]
```diff
@@ -1,45 +1,48 @@
...
+  ppt: https://annakrystalli.me/rrresearchACCE20/
+  git: https://github.com/annakrystalli/rrresearchACCE20
 
 - title: Introduction to Deep Learning
   description: In this full-day introductory workshop, you’ll learn the basics of deep learning by training and deploying neural networks with Python or R.
   pageName: training.md
   level: Core
   topic: 
     - Data
-    - ML
```
Comment (jkwmoore) [inline _data/links/RSE_Courses.yml:18]: If this tag is removed the filter for machine learning will not function. I'm aware there is a technical difference but I don't know if this is a desirable removal.

File: _data/links/RSE_Courses.yml [hunk 2]
```diff
@@ -1,45 +1,48 @@
...
+  ppt: https://rse-shef-hacktoberfest-2020.netlify.app
+  git: https://github.com/RSE-Sheffield/hacktoberfest2020
 
 - title: git & GitHub through GitKraken - from Zero to Hero!
   description: A training course to learn version control and collaboration through Git, GitHub & GitKraken Client.
   pageName: training.md
   level: Core
   topic: 
     - OSS
-    - Git
```
Comment (jkwmoore) [inline _data/links/RSE_Courses.yml:39]: These tags are present for the "Contributing to Open Source Software" event. While there are no current filters for these tags I was expecting them to be retained for consistency with the aforementioned?

**Comment by bobturneruk** [comment]
Hopefully all the tags are back to how they were before I started meddling!

**Comment by bobturneruk** [comment]
I'd prefer to have no `url` if we have no plans to run the course. People can get the material from the `ppt` link. Does this seem OK?

**Comment by jkwmoore** [comment]
Fair enough, with that in mind would https://rse.shef.ac.uk/events/ be preferable for the ones that do have a `url:` ?

**Comment by bobturneruk** [comment]
I'd prefer leaving them blank so people don't go hunting for courses that are not scheduled and may not be run again, but don't feel super strongly about this. Whatever you prefer here.

File: _data/links/RSE_Courses.yml (hunk 2 above)
Comment (jkwmoore) [reply _data/links/RSE_Courses.yml:39]: @bobturneruk Did you want to reinstate these tags for Git / VC?

**Review by jkwmoore — APPROVED** [review]
LGTM - just worth checking whether the Git / VC tags should be reinstated.

**Comment by bobturneruk** [comment]
Hopefully fixed. I need to sort out the colours in my diff tool - the `-` in the `yaml` and the `-` showing a diff have been confusing the flip out of me. Sorry for the confusion!

**Comment by bobturneruk** [comment]
Cheers forgot to merge!

**Pull request merged**